__tests__/chart.test.js

```diff
@@ -0,0 +1,104 @@
+
+            presentation.buildPowerPoint();
+            presentation.setSlideSize(PptxUnitHelper.fromInches(13.33), PptxUnitHelper.fromInches(7.5));
+
+            let slide1 = presentation.getSlide('slide1');
+
+            expect(slide1.content).toBeDefined();
+            expect(slide1.content).not.toBeNull();
+
+            let barChartData = [
+                {
+                    name: 'Series 1',
+                    labels: ['Category 1', 'Category 2', 'Category 3', 'Category 4'],
+                    values: [4.3, 2.5, 3.5, 4.5],
+                },
+                {
+                    name: 'Series 2',
+                    labels: ['Category 1', 'Category 2', 'Category 3', 'Category 4'],
+                    values: [2.4, 4.4, 1.8, 2.8],
+                },
+                {
+                    name: 'Series 3',
+                    labels: ['Category 1', 'Category 2', 'Category 3', 'Category 4'],
+                    values: [2.0, 2.0, 3.0, 5.0],
+                },
+            ];
+
+            await slide1.addChart('bar', barChartData, { x: 100, y: 100, cx: 400, cy: 300 });
+
+            let slide2 = presentation.addSlide();
+
+            barChartData = [
+                {
+                    name: 'Series 1',
+                    color: 'FF0000',
+                    labels: ['Category 1', 'Category 2', 'Category 3', 'Category 4'],
+                    values: [1.3, 4.5, 3.5, 4.5],
+                },
+                {
+                    name: 'Series 2',
+                    color: PPTX.SchemeColors.ACCENT6,
+                    labels: ['Category 1', 'Category 2', 'Category 3', 'Category 4'],
+                    values: [6.4, 2.4, 3.8, 5.8],
+                },
+                {
+                    name: 'Series 3',
+                    labels: ['Category 1', 'Category 2', 'Category 3', 'Category 4'],
+                    values: [1.2, 3.7, 2.5, 1.0],
+                },
+                {
+                    name: 'Series 4',
+                    labels: ['Category 1', 'Category 2', 'Category 3', 'Category 4'],
+                    values: [2.4, 0.8, 0.2, 6.0],
+                },
+                {
+                    name: 'Series 5',
+                    labels: ['Category 1', 'Category 2', 'Category 3', 'Category 4'],
+                    values: [3.0, 4.0, 5.0, 6.0],
+                },
+            ];
+
+            await slide2.addChart('bar', barChartData);
+
+            presentation.save(`${tmpDir}/chart.pptx`);
+            expect(fs.existsSync(`${tmpDir}/chart.pptx`)).toBe(true);
+        } catch (err) {
+            throw err;
+        }
+    });
+
+    test('should be able to add a chart to an existing PowerPoint', () => {
+        console.log('TODO...');
+    });
+});
+
+function prepareTmpDir(dir) {
+    if (!fs.existsSync(dir)) {
+        fs.mkdirSync(dir);
+    } else {
+        emptyDir(dir);
+    }
+}
+
+function emptyDir(dir) {
+    if (fs.existsSync(`${__dirname}/tmp/chart.pptx`)) {
```

Comment:
Because I unzip pptx's into that tmp directory all the time. If I have an unzipped file viewed in ATOM, or doing WinMerge compare, or locked in any way, that function will complain. That's why I changed it to just delete that one file, so I can have other file names open and still run the test.